pkg/apis/pingcap.com/v1alpha1/types.go

```diff
@@ -141,11 +151,13 @@ type TiKVSpec struct {
 	MaxFailoverCount int32  `json:"maxFailoverCount,omitempty"`
 }
 
+// +k8s:openapi-gen=true
```

Comment:
This is not used anymore, so I think it's unnecessary to generate the CRD.

Reply:
fix.